src/main/java/net/sf/jabref/gui/IconTheme.java

```diff
@@ -157,6 +157,8 @@
         FORUM("\uf28c"), /* css: forum */
         FACEBOOK("\uf20c"), /* css: facebook */
         BLOG("\uf46b"), /* css: rss */
+        GLOBAL_SEARCH_ON("\uF1E7"),
+        GLOBAL_SEARCH_OFF("\uF1E8"),
```

Comment:
Can you please add a comment here about the css names? `earth` and `earth-off`

Will make it easier if they start moving the icons around in the font...